classes/search/ArticleSearchDAO.php

```diff
@@ -42,83 +46,65 @@ public function getPhraseResults($journal, $phrase, $publishedFrom = null, $publ
             return [];
         }
 
-        $sqlFrom = '';
-        $sqlWhere = '';
-        $params = [];
+        $q = DB::table('submissions', 's')
+            ->join('journals AS j', 'j.journal_id', '=', 's.context_id')
+            ->leftJoin(
+                'journal_settings AS js',
+                fn (JoinClause $j) => $j
+                    ->on('j.journal_id', '=', 'js.journal_id')
+                    ->where('js.locale', '=', '')
+                    ->where('js.setting_name', '=', 'publishingMode')
+            )
+            ->join('publications AS p', 'p.publication_id', '=', 's.current_publication_id')
+            ->join(
+                'publication_settings AS ps',
+                fn (JoinClause $j) => $j
+                    ->where('ps.setting_name', '=', 'issueId')
+                    ->whereColumn('ps.publication_id', '=', 'p.publication_id')
+                    ->where('ps.locale', '=', '')
+            )
+            ->join('issues AS i', 'i.issue_id', '=', DB::raw('CAST(ps.setting_value AS ' . (DB::connection() instanceof MySqlConnection ? 'UNSIGNED' : 'INTEGER') . ')'))
```

Comment:
The biggest improvement came from this modification.
I'll create another issue to move the `issue_id` to a physical field, I've seen this `CAST()` in several places.